alpha/template/semver/types.go

```diff
@@ -1,19 +1,18 @@
 package semver
 
 import (
+	"context"
 	"io"
 
 	"github.com/blang/semver/v4"
 
-	"github.com/operator-framework/operator-registry/alpha/action/migrations"
-	"github.com/operator-framework/operator-registry/pkg/image"
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
 )
 
 // data passed into this module externally
 type Template struct {
```

Comment:
suggestion: should we just make Template (as defined in basic with just the RenderBundle method) a shared type that basic and semver sub-type and implement? The basic and semver packages could even export a factory functions to get new instances of templates?